src/logic/calculate.js

```diff
@@ -0,0 +1,15 @@
+import Operate from './operate'; /* eslint prefer-const: "error" */
+
+const Calculate = (data = {}, btnName) => {
+  let { total, next } = data;
+  const { operation } = data;
+  if (btnName === '+/-') {
+    total *= -1;
+    next *= -1;
+    return total;
+  }
+
+  return Operate(total, next, operation);
+};
+
+export default Calculate;
```

Comment:
This logic isn't enough...but we dont review the function logic actually, because it will catchup with you on `milestone5`....just drawing your attention

Reply:
I will keep that in mind, thank you for your constructive review :100: